### DISCUSSION TITLE
Reticulum was amazing!

### DISCUSSION THREAD

**Discussion by silveroranges** [markqvist/Reticulum#107] [General]
First real world use of Reticulum for me. I have 5 raspberry pi's with LoRa that you connect to like a wifi router. Hurricane Ian made landfall a few miles from where I live, and knocked out internet, cellular, and power. I decided this would be the perfect real world test of Reticulum, and I distributed all 5 devices to my friends/neighbors in my immediate vicinity (<.5miles) I had power banks for each one, and I put them in baggies because I haven't 3D printed cases yet and it was pretty wet. 

It worked! Everybody was using sideband to communicate with their android phones, and it convinced one apple user to convert over haha. It wasn't without its glitches and bugs, I had to set up the phone apps and change some settings on their phones, but to have communication when nobody else did was awesome. Next time something like this happens I hope to have more nodes set up with permanent locations in my area, so that a person can go to a hotspot to send messages to somebody else, as well as a way to bridge the larger gaps in my area over 15-20 miles. The largest hop in the network was half a mile exactly, and while we had to set the devices up on the highest areas we could find to get a connection and it was slow, it worked. 

On another note, what would you consider the best way to update reticulum is on remote nodes? If I give nodes to neighbors and friends, I would be the one maintaining them and ensuring they are up to date.

**Comment by markqvist** [comment]
Wow, this is just amazing to hear. Thank you so much for sharing it @silveroranges! I feel very thankful and motivated after reading this.

While I am sorry I did not have it ready *before* hurricane season, I think you will really like the next version of Sideband then. So many of the little bugs and annoyances have been sorted out now:

- It now runs Reticulum and LXMF as a proper background service on Android, so it can stay connected to the Reticulum network even when the device is sleeping.
- The UI has been improved quite a bit.
- Identities (cryptographic keys) can be imported and exported.
- There is notification support both on android, linux and mac.
- Lots of little bugs and flaws have been fixed.
- Dark/light mode theme support.
- Much better handling of network and messages "under the hood".

> and it convinced one apple user to convert over haha

This is some crazy kind of praise you don't get everyday. Normally not even **voodoo** will do that!

Regarding updates, I will be building in a easy-to-use update using the base utils included in RNS in one of the next releases, so you can remote update/install over Reticulum only. You _can_ actually already do it with `rncp` and `rnx` (have done so myself to update my car, that was like 15 kilometers away, and only available over Reticulum).

Here is the general idea if you're feeling adventurous now:

```
# Copy RNS distribution to remote
rncp rns-0.3.12-py3-none-any.whl 57048b06aeeb8d8b99e524569dba0905

# Start interactive RNX session with remote
rnx -x 1678179bc15cecc463a1ca0cd9f8d718

# Execute update command over RNX
> pip uninstall -y rns; pip install ./rns-0.3.12-py3-none-any.whl

# Reboot remote system
> reboot
```

Props to you for being so cool and distributing devices in your community! If you need input or help regarding the more permanent setup, I will answer everything as best I can!

**Reply by markqvist** [reply]
Aaaand the next Sideband release is here: https://github.com/markqvist/Reticulum/discussions/110
